carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/ownership/Ownership.java

```diff
@@ -68,15 +71,15 @@ public static String getMethodOwner(ITestResult result, OwnerType type) {
                 if (testMethod.isAnnotationPresent(MethodOwner.class)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.251.139/sonarqube/coding_rules#rule_key=squid%3AS1066)